kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/WifiAccessPointImpl.java

```diff
@@ -141,4 +142,82 @@ public String toString() {
         }
         return sb.toString();
     }
+    
+    @Override
+	public int hashCode() {
```

Comment:
I don't have any problem with the implementation that Eclipse IDE generates for `equals` and `hashcode`. I personally prefer using`Objects` to write less verbose implementations. It's just a syntactic sugar to the code. But if you feel that it would be easier for you to manage this way, I am definitely okay with that 👍 .